odoo/fields.py

```diff
@@ -4321,15 +4321,21 @@ def read(self, records):
         domain = self.get_domain_list(records) + [(inverse, 'in', records.ids)]
         lines = comodel.search(domain)
 
+        # group lines by inverse field (without prefetching other fields)
```

Comment:
the comment should not be moved I think (the method is an utility, but the comment is about the line below)